app/connector/http/pom.xml

```diff
@@ -125,12 +125,7 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>io.syndesis.integration</groupId>
-      <artifactId>integration-runtime</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.hibernate</groupId>
+      <groupId>org.hibernate.validator</groupId>
```

Comment:
This is part of the Hibernate validator upgrade, this should not be on `1.6.x` (#5143)

Reply:
good catch